DATA/MISSIONS/M08/m08.ini

```diff
@@ -586,6 +586,8 @@ Act_StartDialog = meet_more_dragons
 Act_GiveObjList = Ozu, goto_hole
 Act_ActTrig = set_navmarker_to_jumphole
 Act_PlayMusic = music_reveal_and_exposition, none, none, none
+Act_SetNNObj = null, OBJECTIVE_HISTORY
+Act_PobjIdle = no_params
```

Comment:
Use the `stop_player` trigger that I added in an earlier commit (https://github.com/FLHDE/freelancer-hd-edition/commit/c4762e2d1c8ed29a2a87c2469bdf8cb964e50268).